src/arrayMethodJoin.js

```diff
@@ -5,7 +5,34 @@
  */
 function applyCustomJoin() {
   [].__proto__.join2 = function(separator) {
```

Comment:
Якщо не буде сепаратора, то застосує кому, тобто якщо на вході undefined
```suggestion
  [].__proto__.join2 = function(separator = ',') {
```